node/src/components/linear_chain_sync.rs

```diff
@@ -67,6 +73,10 @@ pub use traits::ReactorEventT;
 
 #[derive(DataSize, Debug)]
 pub(crate) struct LinearChainSync<I> {
+    init_hash: Option<BlockHash>,
+    chainspec: Chainspec,
+    started: bool,
```

Comment:
Since fast syncing is going to change wire formats, the rest of this code is going to break.  For now, the prototype is part of this...

Reply:
> So the plan is still to get to a version that supports both slow and fast sync?

Doesn't look like we can support slow sync if we change the wire format to the fetchers.  New nodes would send messages with the current payload and old nodes would not be able to parse them.

> And this is a temporary workaround to keep the linear_chain_sync component functional by just letting fast sync do the work?

We will gradually be transitioning from the slow sync process.